everyvoice/preprocessor/preprocessor.py

```diff
@@ -325,27 +325,35 @@ def print_duration(self):
         seconds %= 60
         return f"{hours}h {minutes}m {seconds}s"
 
-    def report(self, tablefmt="simple"):
+    def report(self, tablefmt="simple", audio_only=False):
         """Print a report of the dataset processing"""
         headers = ["type", "quantity"]
-        table = [
+        audio_1 = [
             ["processed files", self.counters.value("processed_files")],
             [
                 "previously processed files",
                 self.counters.value("previously_processed_files"),
             ],
             ["missing files", self.counters.value("missing_files")],
+        ]
+        others = [
             [
                 "missing symbols",
                 len(self.text_processor.missing_symbols) if self.text_processor else 0,
             ],
             ["skipped processes", self.counters.value("skipped_processes")],
             ["nans", self.counters.value("nans")],
+        ]
+        audio_2 = [
             ["audio_empty", self.counters.value("audio_empty")],
             ["audio_too_short", self.counters.value("audio_too_short")],
             ["audio_too_long", self.counters.value("audio_too_long")],
             ["duration", self.print_duration()],
         ]
+        if audio_only:
+            table = audio_1 + audio_2
+        else:
+            table = audio_1 + others + audio_2
```

Comment:
Question: should the other statistics be placed after the audio ones, instead of between audio_1 and audio_2?